juniper_official/Solutions/Pfe-wedge-detection/chassisd-pattern-set.rule

```diff
@@ -0,0 +1,80 @@
+healthbot {
+    ingest-settings {
+        syslog {
+            pattern CHASSISD_EVENT1 {
+                event-id CHASSISD_EVENT1;
+                filter "fpc%{NUMBER:fpc_number}.*CHASSISD_I2CS_READBACK_ERROR.*Readback error from I2C slave for FPC.*";
+                description "This is transient error caused by parity error detected in the I2C response while reading a device";
+                constant syslog-trigger-color {
+                    value red;
+                }
+                constant syslog-trigger-message {
```

Comment:
Why are we keeping this count only a message? Can we not write this as a rule using the count and then display this in the trigger when the count goes beyond 4?

Reply:
@ConstrainedRandomVar , added trigger based on count